1-raw-data/mailinglists/googlegroups/googlegroups_data.go

```diff
@@ -128,7 +181,8 @@ func getTotalTopics(url string) (totalTopics int, err error){
 	return
 }
 
-// Create list of topic ids grouped by month
+//////////////////////////// NOT USING GOROUTINE ///////////////////////////////////////////////////////////
```

Comment:
@teeler can you look at this code block and the block that says using goroutine starting at 301?  With the one starting at 184 I can pull all 61K topic ids but with the code block at 301 it usually stops loading the full list and I get 10K at best. I've been working with time outs to see if that helps but that still hasn't solved my problem and I also get broken pipes or sometimes it hangs without more detail or erroring out.